### DISCUSSION TITLE
Simple useFetch, Netlify and routeRules

### DISCUSSION THREAD

**Discussion by cfab** [nuxt/nuxt#22613] [Questions] [answered]
I'm confused how to set routeRules and caching on Netlify. 
In my index.vue page I do a simple useFetch call on a server api route that returns some data from airtable.
When deployed on netlify, everything is fine: If I change some data in airtable this is immediately reflected on the site. 

Pbl come when I try to implement routeRules: 
- I've tried to put a rule on my api like this:  '/api/dates': { isr: 20 } or '/api/dates': { swr: 20 }
but then surprisingly, my page will always display data from the time it was deployed... except if if I manually reload the page, but even then when navigating the site, when I come back on my index page I get the "old" data (all this assuming I change some data on airtable after the deployment). 

Am I missing something ? 

I've tried to put the cache on the page itself ('/' : { swr: 20 }) and not on the api, but the resulting behavior is also strange: when navigating the site, data is correct but I do a manual reload of my index page I get the "old" data !

How can I achieve this basic behaviour: a routeRule on my api server route, let's say {swr: 20} and having the correct data displayed on my page ? This would be very useful.

**Comment by AloisSeckar** [comment, marked as answer]
What if you try to provide a `key` option that will be unique each time the request is made?

Something like:
```
useFetch('/api/dates', { key: Date.now().toString() })
```

I would say it is rather a problem of `useFetch` not actually re-fetching, because it thinks the request is still the same. I struggled with this for some time on my first Nuxt project.

**Reply by cfab** [reply]
Thank you !
